### ISSUE TITLE
make db_bench_tool_test failed

### ISSUE THREAD

**Issue by rockeet** [facebook/rocksdb#7341]
### Expected behavior
`make db_bench_tool_test` should success

### Actual behavior
`make db_bench_tool_test` failed

### Steps to reproduce the behavior
run `make db_bench_tool_test` 

```
tools/db_bench_tool_test.cc:69:15: error: no matching function for call to 'LoadLatestOptions'
    ASSERT_OK(LoadLatestOptions(db_path_, FileSystem::Default(),
              ^~~~~~~~~~~~~~~~~
./test_util/testharness.h:40:62: note: expanded from macro 'ASSERT_OK'
  ASSERT_PRED_FORMAT1(ROCKSDB_NAMESPACE::test::AssertStatus, s)
                                                             ^
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19903:36: note: expanded from macro 'ASSERT_PRED_FORMAT1'
  GTEST_PRED_FORMAT1_(pred_format, v1, GTEST_FATAL_FAILURE_)
                                   ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19886:34: note: expanded from macro 'GTEST_PRED_FORMAT1_'
  GTEST_ASSERT_(pred_format(#v1, v1), \
                                 ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19862:52: note: expanded from macro 'GTEST_ASSERT_'
  if (const ::testing::AssertionResult gtest_ar = (expression)) \
                                                   ^~~~~~~~~~
./include/rocksdb/utilities/options_util.h:73:8: note: candidate function not viable: no known conversion from 'std::shared_ptr<FileSystem>' to 'rocksdb::Env *' for 2nd argument
Status LoadLatestOptions(const std::string& dbpath, Env* env,
       ^
./include/rocksdb/utilities/options_util.h:78:8: note: candidate function not viable: no known conversion from 'std::string' (aka 'basic_string<char, char_traits<char>, allocator<char> >') to 'const rocksdb::ConfigOptions' for 1st
      argument
Status LoadLatestOptions(const ConfigOptions& config_options,
       ^
tools/db_bench_tool_test.cc:73:77: error: too few arguments to function call, expected at least 3, have 2
        RocksDBOptionsParser::VerifyDBOptions(DBOptions(opt), loaded_db_opts));
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                               ^
./test_util/testharness.h:40:62: note: expanded from macro 'ASSERT_OK'
  ASSERT_PRED_FORMAT1(ROCKSDB_NAMESPACE::test::AssertStatus, s)
                                                             ^
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19903:36: note: expanded from macro 'ASSERT_PRED_FORMAT1'
  GTEST_PRED_FORMAT1_(pred_format, v1, GTEST_FATAL_FAILURE_)
                                   ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19886:34: note: expanded from macro 'GTEST_PRED_FORMAT1_'
  GTEST_ASSERT_(pred_format(#v1, v1), \
                                 ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19862:52: note: expanded from macro 'GTEST_ASSERT_'
  if (const ::testing::AssertionResult gtest_ar = (expression)) \
                                                   ^~~~~~~~~~
./options/options_parser.h:83:3: note: 'VerifyDBOptions' declared here
  static Status VerifyDBOptions(
  ^
tools/db_bench_tool_test.cc:75:72: error: too few arguments to function call, expected at least 3, have 2
                                                    cf_descs[0].options));
                                                                       ^
./test_util/testharness.h:40:62: note: expanded from macro 'ASSERT_OK'
  ASSERT_PRED_FORMAT1(ROCKSDB_NAMESPACE::test::AssertStatus, s)
                                                             ^
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19903:36: note: expanded from macro 'ASSERT_PRED_FORMAT1'
  GTEST_PRED_FORMAT1_(pred_format, v1, GTEST_FATAL_FAILURE_)
                                   ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19886:34: note: expanded from macro 'GTEST_PRED_FORMAT1_'
  GTEST_ASSERT_(pred_format(#v1, v1), \
                                 ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19862:52: note: expanded from macro 'GTEST_ASSERT_'
  if (const ::testing::AssertionResult gtest_ar = (expression)) \
                                                   ^~~~~~~~~~
./options/options_parser.h:89:3: note: 'VerifyCFOptions' declared here
  static Status VerifyCFOptions(
  ^
tools/db_bench_tool_test.cc:79:74: error: too few arguments to function call, expected at least 3, have 2
        RocksDBOptionsParser::VerifyDBOptions(DBOptions(), loaded_db_opts));
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                            ^
./test_util/testharness.h:41:37: note: expanded from macro 'ASSERT_NOK'
#define ASSERT_NOK(s) ASSERT_FALSE((s).ok())
                                    ^
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:21680:25: note: expanded from macro 'ASSERT_FALSE'
  GTEST_TEST_BOOLEAN_(!(condition), #condition, true, false, \
                        ^~~~~~~~~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:9169:34: note: expanded from macro 'GTEST_TEST_BOOLEAN_'
      ::testing::AssertionResult(expression)) \
                                 ^~~~~~~~~~
./options/options_parser.h:83:3: note: 'VerifyDBOptions' declared here
  static Status VerifyDBOptions(
  ^
tools/db_bench_tool_test.cc:81:73: error: too few arguments to function call, expected at least 3, have 2
                                                     cf_descs[0].options));
                                                                        ^
./test_util/testharness.h:41:37: note: expanded from macro 'ASSERT_NOK'
#define ASSERT_NOK(s) ASSERT_FALSE((s).ok())
                                    ^
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:21680:25: note: expanded from macro 'ASSERT_FALSE'
  GTEST_TEST_BOOLEAN_(!(condition), #condition, true, false, \
                        ^~~~~~~~~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:9169:34: note: expanded from macro 'GTEST_TEST_BOOLEAN_'
      ::testing::AssertionResult(expression)) \
                                 ^~~~~~~~~~
./options/options_parser.h:89:3: note: 'VerifyCFOptions' declared here
  static Status VerifyCFOptions(
  ^
tools/db_bench_tool_test.cc:109:13: error: no matching function for call to 'PersistRocksDBOptions'
  ASSERT_OK(PersistRocksDBOptions(DBOptions(opt), {"default"},
            ^~~~~~~~~~~~~~~~~~~~~
./test_util/testharness.h:40:62: note: expanded from macro 'ASSERT_OK'
  ASSERT_PRED_FORMAT1(ROCKSDB_NAMESPACE::test::AssertStatus, s)
                                                             ^
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19903:36: note: expanded from macro 'ASSERT_PRED_FORMAT1'
  GTEST_PRED_FORMAT1_(pred_format, v1, GTEST_FATAL_FAILURE_)
                                   ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19886:34: note: expanded from macro 'GTEST_PRED_FORMAT1_'
  GTEST_ASSERT_(pred_format(#v1, v1), \
                                 ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19862:52: note: expanded from macro 'GTEST_ASSERT_'
  if (const ::testing::AssertionResult gtest_ar = (expression)) \
                                                   ^~~~~~~~~~
./options/options_parser.h:36:8: note: candidate function not viable: no known conversion from 'rocksdb::Env *' to 'rocksdb::FileSystem *' for 5th argument
Status PersistRocksDBOptions(const DBOptions& db_opt,
       ^
./options/options_parser.h:40:8: note: candidate function not viable: requires 6 arguments, but 5 were provided
Status PersistRocksDBOptions(const ConfigOptions& config_options,
       ^
tools/db_bench_tool_test.cc:132:13: error: no matching function for call to 'PersistRocksDBOptions'
  ASSERT_OK(PersistRocksDBOptions(DBOptions(opt), {"default"},
            ^~~~~~~~~~~~~~~~~~~~~
./test_util/testharness.h:40:62: note: expanded from macro 'ASSERT_OK'
  ASSERT_PRED_FORMAT1(ROCKSDB_NAMESPACE::test::AssertStatus, s)
                                                             ^
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19903:36: note: expanded from macro 'ASSERT_PRED_FORMAT1'
  GTEST_PRED_FORMAT1_(pred_format, v1, GTEST_FATAL_FAILURE_)
                                   ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19886:34: note: expanded from macro 'GTEST_PRED_FORMAT1_'
  GTEST_ASSERT_(pred_format(#v1, v1), \
                                 ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19862:52: note: expanded from macro 'GTEST_ASSERT_'
  if (const ::testing::AssertionResult gtest_ar = (expression)) \
                                                   ^~~~~~~~~~
./options/options_parser.h:36:8: note: candidate function not viable: no known conversion from 'rocksdb::Env *' to 'rocksdb::FileSystem *' for 5th argument
Status PersistRocksDBOptions(const DBOptions& db_opt,
       ^
./options/options_parser.h:40:8: note: candidate function not viable: requires 6 arguments, but 5 were provided
Status PersistRocksDBOptions(const ConfigOptions& config_options,
       ^
tools/db_bench_tool_test.cc:155:13: error: no matching function for call to 'PersistRocksDBOptions'
  ASSERT_OK(PersistRocksDBOptions(DBOptions(opt), {"default"},
            ^~~~~~~~~~~~~~~~~~~~~
./test_util/testharness.h:40:62: note: expanded from macro 'ASSERT_OK'
  ASSERT_PRED_FORMAT1(ROCKSDB_NAMESPACE::test::AssertStatus, s)
                                                             ^
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19903:36: note: expanded from macro 'ASSERT_PRED_FORMAT1'
  GTEST_PRED_FORMAT1_(pred_format, v1, GTEST_FATAL_FAILURE_)
                                   ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19886:34: note: expanded from macro 'GTEST_PRED_FORMAT1_'
  GTEST_ASSERT_(pred_format(#v1, v1), \
                                 ^~
third-party/gtest-1.8.1/fused-src/gtest/gtest.h:19862:52: note: expanded from macro 'GTEST_ASSERT_'
  if (const ::testing::AssertionResult gtest_ar = (expression)) \
                                                   ^~~~~~~~~~
./options/options_parser.h:36:8: note: candidate function not viable: no known conversion from 'rocksdb::Env *' to 'rocksdb::FileSystem *' for 5th argument
Status PersistRocksDBOptions(const DBOptions& db_opt,
       ^
./options/options_parser.h:40:8: note: candidate function not viable: requires 6 arguments, but 5 were provided
Status PersistRocksDBOptions(const ConfigOptions& config_options,
       ^
```

**Comment by jay-zhuang** [comment]
Seems related to #6389

**Comment by mrambacher** [comment]
Fixed by #7344
